src/components/common/Input/Input.tsx

```diff
@@ -19,7 +20,8 @@ interface IInput {
   id?: string;
   for?: string;
```

Comment:
what's the point of this prop?

Reply:
Remuved

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/35d1051dd92f9591600c90270a824d4eec0360e7#diff-e2eec15dd4a882dc8cd779ef821494357bd0dface0e4d8e40fd8d7c4f8bbe31a